openshift-install-powervs

```diff
@@ -64,9 +64,11 @@ ARTIFACTS_VERSION=${ARTIFACTS_VERSION:-"release-$RELEASE_VER"}
 #ARTIFACTS_VERSION="v4.5.3"
 #ARTIFACTS_VERSION="master"
 
-TF='./terraform'
-CLI_PATH='./ibmcloud'
-OC_PATH='./oc'
+INSTALL_DIR=$PWD
```

Comment:
Should this be just $PWD or safer to have something like $PWD/install_dir ?

Reply:
Does not matter.. $PWD/install_dir is also in $PWD if you are concerned about using it with docker volume. :-) 

$PWD is good here IMO as user gets noticed which new files gets added to current working directory. But yeah.. this variable can be used in future in case we want user to decide any path as a install directory.

Reply:
my worry is this will pollute the current directory if it's not empty. And I'm not sure if we should rely on the fact that the current directory will be empty

Reply:
These files are meant to be used by the script and provide a way for users to to supply it themself before running the script. If the script don't find it usable it will make it usable.